src/main/webapp/resources/js/pages/UserGroupsPage/components/UserGroupDetailsPage.jsx

```diff
@@ -0,0 +1,150 @@
+import React, { useEffect, useReducer, useRef } from "react";
```

Comment:
File comments

Reply:
Updated in 96f283c7fc96e970e49af9238c991bd38da114d0